com.ca.lsp.cobol/lsp-core-cobol-parser/pom.xml

```diff
@@ -242,6 +242,11 @@
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Do we need to apply any TPSR process in order to introduce this dependency?

Reply:
that's a good point, gonna check on it